src/System.Net.Security/src/win/project.json

```diff
@@ -1,31 +1,31 @@
 {
```

Comment:
I don't see any platform specific dependencies here. Does it make sense to join both unix and win project.json files into one?